src/DynamoCore/Models/DynamoModelEvents.cs

```diff
@@ -303,6 +304,10 @@ internal void OnRequestBugReport()
         /// <param name="e">The event data</param>
         public virtual void OnEvaluationCompleted(object sender, EvaluationCompletedEventArgs e)
         {
+
+            var compileTask = new ExpressionCompileTask(Scheduler, CurrentWorkspace, ExpressionCompilerRegistry);
```

Comment:
This is just here for testing. It executes the compilation task whenever execution is complete. In the future, this task would replace the `UpdateGraphAsyncTask`.